src/cqe.rs

```diff
@@ -64,6 +70,29 @@ impl<'a> CompletionQueueEvent<'a> {
         CompletionQueueEvent { ring, cqe }
     }
 
+    /// Check whether this event is a timeout.
```

Comment:
Let's make this test pass even without timeout support like we discussed earlier.